src/box/memtx_hash.c

```diff
@@ -436,10 +436,24 @@ memtx_hash_index_create_iterator(struct index *base, enum iterator_type type,
 			light_index_iterator_begin(&index->hash_table, &it->iterator);
 			it->base.next = hash_iterator_ge;
 		}
+		/**
+		 * This iterator needs to be supported as a legacy.
+		 * The reason for tracking the gap is the same as below.
+		 */
+		memtx_tx_track_gap(in_txn(), space_by_id(it->base.space_id),
+				   index->base.def->iid, NULL, type,
```

Comment:
Please rebase carefully to actual master. That's an old signature.

Reply:
Rebased & fixed, thanks.